app/_components/Home/Hero/index.jsx

```diff
@@ -1,9 +1,31 @@
-// TODO: Create a Hero component.
-export default function Hero() {
-    return (
-      <main>
-        <h1 className="text-6xl font-bold">Hero</h1>
+// pages/index.js
+import Image from 'next/image';
+import HackathonImage from '../Hero/AIBot.png';
+
+export default function Home() {
+  return (
+    <div className="flex items-center justify-center min-h-screen bg-gray-100 px-4">
+      <main className="flex flex-col items-center justify-center text-center bg-gradient-to-r from-[#E6F2FF] via-[#B3D7FF] to-[#FFFFFF] rounded-3xl shadow-lg p-6 md:p-10 max-w-screen-lg w-full">
```

Comment:
Remove the shadow according to the Figma design.

Reply:
BREAKING CHANGES : Resolved!